lib/conduit_async_ssl.mli

```diff
@@ -33,7 +33,7 @@ val ssl_connect :
   ?verify:(Ssl.Connection.t -> bool Deferred.t) ->
   Reader.t ->
   Writer.t ->
-  (Reader.t * Writer.t) Deferred.t
+  (Reader.t * Writer.t) Deferred.Or_error.t
```

Comment:
Are you sure about `Pipe.create`? I'm pretty sure pipes aren't tied to any particular fd's. It's readers/writers that are fd wrappers.

But anyway if you're reasonably certain then I support this change.